webhonc/src/index.ts

```diff
@@ -0,0 +1,94 @@
+import { zValidator } from "@hono/zod-validator";
+import { Hono } from "hono";
+import { z } from "zod";
+import type { Bindings, Variables } from "./types";
+import { WebHonc } from "./webhonc";
+
+const app = new Hono<{ Bindings: Bindings; Variables: Variables }>();
+
+app.get("/ws", async (c) => {
+  if (c.req.header("upgrade") !== "websocket") {
+    return new Response("Not a websocket request", { status: 426 });
+  }
+
+  const id = c.env.WEBHONC.newUniqueId();
```

Comment:
question: what happens to old ids? are they still persisted somewhere? should we dispose of them when the websocket connection dies?

Reply:
they get disposed in Cloudflare's runtime I believe. At least there's no documentation on clean up work needed AFAIK